Documentation/ApiOverview/Events/Events/FrontendLogin/_LogoutConfirmedEvent/_DeletePrivateKeyOnLogout.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace MyVendor\MyExtension\EventListener;
+
+use MyVendor\MyExtension\KeyPairHandling\KeyFileService;
+use TYPO3\CMS\Core\Attribute\AsEventListener;
+use TYPO3\CMS\Core\Context\Context;
+use TYPO3\CMS\Core\Context\UserAspect;
+use TYPO3\CMS\Core\Type\ContextualFeedbackSeverity;
+use TYPO3\CMS\FrontendLogin\Event\LogoutConfirmedEvent;
+
+/*
+TODO: Comment this in once you drop TYPO3 v12.5 support
+#[AsEventListener(
```

Comment:
I suggest to remove it. You don't have to use the attribute in v13, you can. The tagging in Services.yaml still works in v13 and will work in future versions.